src/components/new-request/search.vue

```diff
@@ -765,12 +766,9 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
    */
   async actionNowClicked () {
     const legalType = this.entityTypeToCorpType(this.entity_type_cd)
-    if (this.getIsAuthenticated) {
-      if (this.isConversion) {
+    if (this.isAuthenticated) {
+      if (this.isConversion || this.isRestoration) {
         this.goToEntityDashboard(this.business.identifier)
-      } else if (this.isRestoration) {
-        const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
-        Navigate(`${dashboardUrl}${this.business.identifier}`)
```

Comment:
These 2 lines already existed in `this.goToEntityDashboard()` so I updated line 770 >>>